substrate/frame/bags-list/src/lib.rs

```diff
@@ -263,6 +292,12 @@ pub mod pallet {
 	pub(crate) type ListBags<T: Config<I>, I: 'static = ()> =
 		StorageMap<_, Twox64Concat, T::Score, list::Bag<T, I>>;
 
+	/// State of the lock for implicit and explicit ordering of the IDs in the bags list.
+	///
+	/// When the lock is set, no item re-ordering should happen.
+	#[pallet::storage]
+	pub(crate) type LockOrder<T: Config<I>, I: 'static = ()> = StorageValue<_, (), OptionQuery>;
```

Comment:
We already have this information in epm pallet, and we could get this via some new trait.